packages/alfa-style/src/property/border-bottom-left-radius.ts

```diff
@@ -0,0 +1,32 @@
+import { Keyword, Length, Percentage } from "@siteimprove/alfa-css";
+import { Parser } from "@siteimprove/alfa-parser";
+
+import { Property } from "../property";
+
+const { pair } = Parser;
+
+/**
+ * @internal
+ */
+
+export type Specified = [Length<"px"> | Percentage, Length<"px"> | Percentage];
+
+/**
+ * @internal
+ */
+export type Computed = Specified;
```

Comment:
```suggestion
export type Specified = [horizontal: Length<"px"> | Percentage, vertical: Length<"px"> | Percentage];
```

At computed value time, lengths are resolved to `px` values.